cmd/local_client_test.go

```diff
@@ -49,7 +49,7 @@ func TestClient_RunNodeShowsEnv(t *testing.T) {
 	assert.Contains(t, logs, "JSON_CONSOLE: false")
 	assert.Contains(t, logs, "ROOT: /tmp/chainlink_test/")
 	assert.Contains(t, logs, "CHAINLINK_PORT: 6688\\n")
-	assert.Contains(t, logs, "ETH_URL: ws://")
+	assert.Contains(t, logs, "ETH_WS_URL: ws://")
```

Comment:
This is a breaking change. Can we stick with ETH_URL, for now and maybe move it in a subsequent PR?

Reply:
Okay, I can revert this and use an override in the places where it's used inconsistently.